spec/models/sign_in/o_auth_session_spec.rb

```diff
@@ -179,4 +179,13 @@
       end
     end
   end
+
+  describe '#user_attributes_hash' do
+    subject { oauth_session.user_attributes_hash }
```

Comment:
This spec is doing a bit more circular validation than I'd like. What I'd do is add the `user_attributes` as fields `oauth_session` object created above, and then you can separately construct the user_attributes hash in a `let` as you'd expect and compare it in the expectation